go/libraries/doltcore/sqle/logictest/DoltRegressionsJenkinsfile

```diff
@@ -0,0 +1,91 @@
+pipeline {
+    agent none
+    stages {
+        stage('TestSQL') {
+            parallel {
+                stage ("update Liquidata/dolt-sql-performance") {
+                    agent {
+                        kubernetes {
+                            label "liquidata-inc-ld-build"
+                        }
+                    }
+                    environment {
+                        PATH = "${pwd()}/.ci_bin:${pwd()}/.ci_bin/node_modules/.bin:${env.PATH}"
+                        DOLT_VERSION = sh(returnStdout: true, script: "(git rev-parse HEAD | tr -d '\n')")
+                        TMPDIR = "${pwd()}/tempDir"
+                        CREDSDIR = "/home/jenkins/.dolt/creds"
+                        DOLT_CREDS = credentials("system-account-dolthub-creds")
+                        TEST_N_TIMES = 5 // like to pass this from the Jenkins UI?
+                        DOLT_GLOBAL_CONFIG = "/home/jenkins/.dolt/config_global.json"
+                    }
+                    steps {
+                        dir ("sqllogictest") {
+                            git url: "https://github.com/liquidata-inc/sqllogictest.git"
+                        }
+                        dir ("go") {
+                            sh "rm -rf $TMPDIR && mkdir $TMPDIR"
+                            sh "go get -mod=readonly ./..."
+                            sh "go build -mod=readonly -o ../.ci_bin/dolt ./cmd/dolt/."
+                        }
+                        sh "rm -rf dolt-sql-performance && dolt clone Liquidata/dolt-sql-performance"
+                        dir ("go/libraries/doltcore/sqle/logictest/main") {
+                            sh "dolt config --global --add user.name 'LiquidataSystemAccount'"
+                            sh "dolt config --global --add user.email 'liquidatasys@gmail.com'"
```

Comment:
I attempted to set `DOLT_GLOBAL_CONFIG = credentials("system-account-dolthub-config")` and just `cat $DOLT_GLOBAL_CONFIG > /home/jenkins/.dolt/config_global.json`, but this broke the config... Im not sure if the encrypted config actually works, may need to be re-encrypted....